redaxo/src/core/lib/util/markdown.php

```diff
@@ -136,14 +136,12 @@ private function handleHeader(array $block = null)
 
         [$level] = sscanf($block['element']['name'], 'h%d');
 
-        if ($level < $this->topLevel || $level > $this->bottomLevel) {
-            return $block;
-        }
+        $displayHeader = $level >= $this->topLevel && $level <= $this->bottomLevel;
```

Comment:
Braucht es die Variable überhaupt? Kann die Bedinung nicht direkt in das `if` unten?

Reply:
Nein, kann direkt ins if. 🤦‍♂️